flink-libraries/flink-table/src/test/scala/org/apache/flink/table/runtime/stream/table/TableSourceITCase.scala

```diff
@@ -778,4 +778,42 @@ class TableSourceITCase extends AbstractTestBase {
     assertEquals(expected.sorted, StreamITCase.testResults.sorted)
   }
 
+  @Test
+  def testTableSourceScanWithConversion(): Unit = {
```

Comment:
Rename to `testTableSourceScanWithPermutedFields()`?

Reply:
Agree! Will rename this test.